09-schema/01-concepts.md

```diff
@@ -751,16 +751,90 @@ TypeQLDefine query = TypeQL.define(
 [tab:end]
 </div>
 
-## Undefine
-As the name suggests, we use the `undefine` keyword to remove the definition of a type or its association with other types from the schema.
+## Modify an existing schema
+We can modify an existing schema by adding or deleting concepts: entities, relations, attributes and rules.
+
+To add we use `define` and to delete an existing concept we use `undefine`. 
+
+We can't modify concepts in a schema directly. To modify an existing concept in a schema (e.g. rename something) we 
+shall delete the old concept (by using [undefine](#deleting-from-a-schema-with-undefine)) and add the new one (by using 
+[define](#adding-to-a-schema-with-define)).
+
+Don't forget to modify all references accordingly (with deletion and adding).
+
+If there are a lot of changes to be done consider creating a brand-new database to load a new schema from scratch.
+
+### Adding to a schema with Define
+
+To add concepts to an existing schema we need to do a query with a single `define` statement at the beginning, just like
+for a new schema. For example, lets add attribute `phone-number` to the `person` entity:
+
+<div class="tabs dark">
+
+[tab:TypeQL]
+```typeql
+define
+
+person owns phone-number;
+```
+[tab:end]
+
+[tab:Java]
+```java
+TypeQLDefine query = TypeQL.define(
+        type("person").owns("phone-number")
+        );
+```
+</div>
+
+Any concepts that already exist in a schema you are trying to modify will not be added to prevent doubling. That is why 
```

Comment:
doubling -> duplication

This applies everywhere else we've used the word 'doubling'. Not necessarily wrong, just more familiar language to most developers imo.

Reply:
I think you are right here. Fixed it everywhere I found it  https://github.com/vaticle/docs/pull/652/commits/85020f7fcfe6cf2e6378f3e1e7c6d38cfd9b10b7